project/ren-yi-wang-david/README.rst

```diff
@@ -0,0 +1,106 @@
+==============================================================
+FTST
+==============================================================
+
+Basic Information
+=================
+FDM Thermal Simulator Tool
+
+GitHub repository: https://github.com/ren-yi-wang-david/FTST
+
+Problem to Solve
+================
+power induced thermal distribution,steady state heat transfer,etc.
+
+Prospective Users
+=================
+this tool is developed for CAE,VLSI,CFD simulation user providing verification ,early stage prediction,thermal-aware co optimization.
+
+System Architecture
+===================
+using FDM to discretize the governing equation of heat transfer
+
+using  LU decomposition for solver
+
+using python for visualization
+
+Using C++ to implement the algorithm.
+
+Using pybind11 to wrap c++ in python.
+
+Using pybind11 to make API in python.
+
+C++: kernel system
+
+Python: user interface
+
+an user-friendly UI
+
+wrap the kernel system by using pybind11
+
+workflow:
+===============
+Input: Massive matrices(.txt) for power map 
+
+Output:temperature distribution(.txt)and visualization output(heatmap)
+
+API Description
+===============
+(python)
+from FTST import ThermalSolver
+
+solver = ThermalSolver(nx=50, ny=50, k=150.0)
+solver.set_boundary("left", 300)
+solver.set_boundary("right", 350)
+solver.set_power_map("power_map.txt")
+
+solver.run("LU")
+solver.plot("heatmap.png")
+solver.write_csv("temperature.txt")
+
+Engineering Infrastructure
+==========================
+license:MIT
+
+Testing framework: Google Test (C++), pytest (Python)
+
+Compiler: g++, python
+
+Build System: make
+
+Version Control: git
+
+Automatic build system: GNU make
+
+Documentation: README.md
+
+Programming languages: Python (visualization), C++ (computation)
+
+golden:ansys icepak
+
+Schedule
+========
+
+* Planning phase (6 weeks from 10/20 to 12/8):
```

Comment:
Developing FDM with matrix inversion can be a lot of work.  You may start early.